iroha-lib/model/Query.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef QUERY_HPP
+#define QUERY_HPP
+
+#include "LibsAndClassDeclarations.h"
+
+#include "crypto/keypair.hpp"
+#include "queries.pb.h"
+#include "generators/QueryGenerator.hpp"
+
+
+namespace iroha_lib
+{
+
+class Query
+{
+    uint64_t counter_;
```

Comment:
Do not rely on default visibility. Explicitly set this as `private`.

Reply:
Added.